README.md

```diff
@@ -1 +1,170 @@
+
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [🧑🏻‍💻 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Clone](#clone)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Vet Clinic Database - PostgreSQL Microverse Project<a name="about-project"></a>
+
+> Vet Clinic Database - PostgreSQL database developed with PostgreSQL
+In this project, we have use a relational database to create the data structure for a vet clinic. We started with one table, and step by step, you will create a complete database with data about:
+
+animals
+animals' owners
+clinic employees
+visits
+Apart from building data structure, you will be challenged with tasks that require SQL queries to answer specific questions.
+
+How to build your database
+You will build your database in your local environment, and Microverse will not require you to share it. Therefore only you will have access to its data (which is a good thing from the security point of view). However, you will need to share all SQL queries you use to build and query your database to get feedback about them.
+
+How to share your projects with a code reviewer
+You will share your SQL queries as text files hosted in a GitHub repository. It is mandatory to use the curriculum-databases-projects-template to initialize the repository. You should keep the structure of files from the template. You will use the regular code review process for your projects.
+
+Projects list
+You will be building your "Vet clinic" database for the whole week. Here is the list of projects that will guide you through the steps described above. You will find details about each of the project requirements in the upcoming program activities.
+
+Project 1: create animals table.
+Project 2: query and update animals table.
+Project 3: query multiple tables.
+Project 4: add "join table" for visits.
+Project 5: database performance audit.
+Exercise: add database schema diagram.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Animals Table**
+- **Species Table**
+- **Owners Table**
+- **Multiple Queries**
+- **Join Queries**
+- **Use Transaction**
+- **Use Relations One to Many & Many to Many**
+- **DB Schema**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🧑🏻‍💻 Live Demo <a name="live-demo"></a>
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove the Live Demo section because you don't have any working live link for your app 😨 KIndly remove this section to make your readme descriptive and professional 👍